packages/aws-cdk-lib/aws-appsync/README.md

```diff
@@ -391,21 +391,33 @@ secondApi.addNoneDataSource('SecondSourceDS', {
 });
 
 // Merged API
-new appsync.GraphqlApi(this, 'MergedAPI', {
+const mergedApi = new appsync.GraphqlApi(this, 'MergedAPI', {
   name: 'MergedAPI',
   definition: appsync.Definition.fromSourceApis({
     sourceApis: [
       {
         sourceApi: firstApi,
         mergeType: appsync.MergeType.MANUAL_MERGE,
       },
-      {
-        sourceApi: secondApi,
-        mergeType: appsync.MergeType.AUTO_MERGE,
-      },
     ],
   }),
 });
+
+
+// An additional way of declaring a source api association using the SourceApiAssociation construct. This is useful when a source api is deployed in a different stack from the Merged API.
+// The source api owner can associate it to the Merged API with this construct.
```

Comment:
Rather than updating the previous example, I'd rather you make this into a new use-case driven section with a fully fresh example. Perhaps named something like "merging APIs from different stacks".